synapse/axon.py

```diff
@@ -32,6 +32,12 @@
 
 CHUNK_SIZE = 16 * s_const.mebibyte
 
+def _path_sanitize(blobstorpath):
+    '''
+    The path might contain username/password, so just return the last part
+    '''
+    return '.../' + blobstorpath.rsplit('/', 1)[-1]
```

Comment:
Since this url should be telepath or aliases friendly; we should be able to run it through url chop and recreate the url without creds in it